hook.py

```diff
@@ -1,10 +1,24 @@
 import logging
+import asyncio
 from pathlib import Path
 from subprocess import Popen
+from ssl import get_server_certificate
 
 name = 'SSL'
 description = 'Run an SSL proxy in front of the server'
-address = None
+address = ''
+
+async def _check_cert():
+    await asyncio.sleep(5)
+    default_cert = ''
+    server_cert = ''
+    server_cert = get_server_certificate(('127.0.0.1', 8443))
+    with open(Path('plugins/ssl/conf/insecure_certificate.pem'), 'r') as f:
+        default_cert = f.read()
+    if server_cert == default_cert:
```

Comment:
nicely done figuring out this function though if we move the key back into the same pem file (see my other comment), then we'll need to make a tweak on this line since the values won't be equal anymore: 

```python
if server_cert in default_cert:
```